packages/calypso-products/src/constants/akismet.ts

```diff
@@ -95,4 +110,8 @@ export const AKISMET_UPGADES_PRODUCTS_MAP = {
 	[ PRODUCT_AKISMET_PLUS_40K_YEARLY ]: `/checkout/akismet/${ PRODUCT_AKISMET_ENTERPRISE_YEARLY }`,
 	[ PRODUCT_AKISMET_ENTERPRISE_MONTHLY ]: 'https://akismet.com/enterprise',
 	[ PRODUCT_AKISMET_ENTERPRISE_YEARLY ]: 'https://akismet.com/enterprise',
+	[ PRODUCT_AKISMET_ENTERPRISE_350K_MONTHLY ]: 'https://akismet.com/enterprise',
+	[ PRODUCT_AKISMET_ENTERPRISE_350K_YEARLY ]: 'https://akismet.com/enterprise',
+	[ PRODUCT_AKISMET_ENTERPRISE_2M_MONTHLY ]: 'https://akismet.com/enterprise',
+	[ PRODUCT_AKISMET_ENTERPRISE_2M_YEARLY ]: 'https://akismet.com/enterprise',
```

Comment:
I figured any plan that can possibly be upgraded should still have an upgrade button, so I added these enterprise plans with an upgrade button that will go to `/enterprise` excluding the top enterprise plans ( GT2M yearly/monthly )